core/trino-main/src/test/java/io/trino/block/BlockAssertions.java

```diff
@@ -771,11 +789,19 @@ public static RunLengthEncodedBlock createRLEBlock(long value, int positionCount
         return chooseRandomUnique(positionCount, nullCount);
     }
 
-    private static Set<Integer> chooseRandomUnique(int bound, int count)
```

Comment:
why this change. Please undo

Reply:
> I would still leave the fast path though as my heart breaks when 8k numbers have to be generated and shuffled just to take the first two...

IMO it's not really important since it's not a perf critical code so simplicity > optimizations